tiledb/sm/array_schema/array_schema.cc

```diff
@@ -333,6 +333,8 @@ unsigned int ArraySchema::dim_num() const {
 }
 
 void ArraySchema::dump(FILE* out) const {
+  if (out == NULL)
```

Comment:
We generally use `nullptr` instead of `NULL` .

Reply:
Just like everybody else I prefer `nullptr` and meant to ask.  And some point I "looked around" also for use in the R package and saw `NULL` used somewhere so I ran with it.  Will fix on both ends.  `nullptr` is uniformly better.